docs/Components/BaseComponent.md

```diff
@@ -83,3 +83,18 @@ This property returns a dictionary with the hierarchical relationship of each co
 print(base_component.nested_components)
 >>> {'component': BaseComponent(name='component', type='generic_component', mass=3.4, children={'wing1': BaseComponent(name='wing1', type='wing', mass=1.1, children={}, external_forces={}, external_moments={})}, external_forces={}, external_moments={}), 'wing1': BaseComponent(name='wing1', type='wing', mass=1.1, children={}, external_forces={}, external_moments={})}
 ```
+## add_external_force_function
+This function is responsible for adding an external force applied to a given point on the component
```

Comment:
This *method is responsible for *appending an external force *function to the component. *The force function returns a force vector and an application point.